doc/DESI_SPECTRO_DATA/NIGHT/EXPID/desi-EXPID.rst

```diff
@@ -16,138 +16,302 @@ There is one HDU per spectrograph camera with EXTNAMEs like
 B0, B1, ... R0, R1, ... Z8, Z9.  The structure of each of these is
 the same; only one is explicitly documented below.
 
-====== ======= ===== ===================
-Number EXTNAME Type  Contents
-====== ======= ===== ===================
-HDU0_  PRIMARY IMAGE Blank except for header keywords
-HDU1_  B0      IMAGE Raw data from the b0 spectrograph
-====== ======= ===== ===================
+================= ========= ======== ====================================
+Number            EXTNAME   Type     Contents
+================= ========= ======== ====================================
+HDU0_             SP0       IMAGE    Blank except for header keywords
+HDU1_             SPECTCON0 BINTABLE Telemetry data
```

Comment:
Until the FV tests started, I hadn't realized that Klaus has one format for single spectrograph readouts and a different but closely related format for multi-spectrograph readouts.  Let's keep this datamodel file focused on what the final multi-spectrograph format should be, and then work with ICS to see if they would be willing to make the single-spectrograph case a simple subset of that (instead of a slightly different format).

Single spectrograph | Multi-spectrograph | pixsims (so far)
--- | --- | ---
HDU0 = SP0 .. SP9 | HDU0 = SPS | HDU0 = (no EXTNAME)
HDU1 = SPECTCON0 .. SPECTCON9 | last HDU = SPECCONS | not in pixsims
filename sp{n}-*.fits.fz | filename desi-*.fits.fz | filename desi-*.fits.fz

Reply:
I agree that this data model should be focused on the final multi-spectrograph file.  But what I have to *work* with to get all the details of the header keywords are the single spectrograph files.  One does what one must...